package.json

```
@@ -275,7 +275,7 @@
     "@metamask/accounts-controller": "^11.0.0",
     "@metamask/address-book-controller": "^4.0.1",
     "@metamask/announcement-controller": "^6.1.0",
-    "@metamask/approval-controller": "^6.0.0",
+    "@metamask/approval-controller": "^6.0.1",
```

(could) bump to newer 6.0.2 - also the same.
```suggestion
    "@metamask/approval-controller": "^6.0.2",
```

---

True, we could include it. But since [the target is Release 125](https://github.com/MetaMask/metamask-extension/issues/24140), and since I feel lucky that the CI went through without flaky tests failing this time, I'm inclined to merge this and bump the package again in another PR